test/http_to_do_test.exs

```diff
@@ -39,4 +47,74 @@ defmodule HTTPServerTest.ToDo do
 
     assert API.create(handle_resp) == error_message
   end
+
+  test "Can delete \"{\"todo1\":\"Act\"}\" from data file" do
+    todo_to_be_deleted = "1"
+
+    API.delete(todo_to_be_deleted)
+
+    file_contents =
+      File.read!(@file_path)
+      |> JSON.decode!()
+
+    refute file_contents[todo_to_be_deleted]
+  end
+
+  test "Can delete \"{\"todo4\":\"Teardown\"}\" from data file" do
+    todo_to_be_deleted = "4"
+
+    API.delete(todo_to_be_deleted)
+
+    file_contents =
+      File.read!(@file_path)
+      |> JSON.decode!()
+
+    refute file_contents[todo_to_be_deleted]
+  end
+
+  test "The file will not change when id = 5" do
+    todo_to_be_deleted = "5"
+
+    data_before_delete =
+      File.read!(@file_path)
+      |> JSON.decode!()
+
+    API.delete(todo_to_be_deleted)
+
+    data_after_delete =
+      File.read!(@file_path)
+      |> JSON.decode!()
+
+    assert data_before_delete == data_after_delete
+  end
+
+
+  test "The file will not change when id = 0.5" do
+    todo_to_be_deleted = "0.5"
+
+    data_before_delete =
+      File.read!(@file_path)
+      |> JSON.decode!()
+
+    API.delete(todo_to_be_deleted)
+
+    data_after_delete =
+      File.read!(@file_path)
+      |> JSON.decode!()
+
+    assert data_before_delete == data_after_delete
+  end
+
+  test "Returns an empty map when no todos have been created." do
```

Comment:
This is an example of good test name. (ref: [comment](https://github.com/kelseyroy/http_server/pull/34/files#r1269841959))

Reply:
Though it still talks about "maps", and that is an implementation detail. 
```suggestion
  test "Is empty when no todos have been created." do
```